src/Aquifer.Jobs/Managers/SendResourceAssignmentNotifications.cs

```diff
@@ -0,0 +1,101 @@
+using Aquifer.Common.Services;
+using Aquifer.Data;
+using Aquifer.Data.Enums;
+using Aquifer.Jobs.Configuration;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Aquifer.Jobs.Managers;
+
+public class SendResourceAssignmentNotifications(
+    IOptions<ConfigurationOptions> _configurationOptions,
+    AquiferDbContext _dbContext,
+    IEmailService _emailService,
+    ILogger<SendResourceAssignmentNotifications> _logger)
+{
+    private const string _everyTenMinutesCronSchedule = "0 */10 * * * *";
+    private const string _tenSecondDelayInterval = "00:00:10";
+
+    [Function(nameof(SendResourceAssignmentNotifications))]
+    [FixedDelayRetry(maxRetryCount: 1, delayInterval: _tenSecondDelayInterval)]
+#pragma warning disable IDE0060 // Remove unused parameter: A (non-discard) TimerInfo parameter is required for correct Azure bindings.
+    public async Task RunAsync([TimerTrigger(_everyTenMinutesCronSchedule)] TimerInfo timerInfo, CancellationToken ct)
+#pragma warning restore IDE0060 // Remove unused parameter
+    {
+        var jobHistory = await _dbContext.JobHistory
+            .AsTracking()
+            .SingleAsync(jh => jh.JobId == JobId.SendResourceAssignmentNotifications, ct);
+
+        // Possible Improvement: Only send notifications when a user is assigned to the *most recent* ResourceContentVersion.
+        var userHistories = await _dbContext.ResourceContentVersionAssignedUserHistory
```

Comment:
How does this handle the situation where a user shouldn't receive an email on assignment? For example, project start, the company lead shouldn't get an email for the items assigned, because they just get the project started email?